.ci/onnx/jenkins/Jenkinsfile

```diff
@@ -30,6 +30,25 @@ GIT_COMMIT_AUTHOR_EMAIL=""
 GIT_COMMIT_HASH=""
 GIT_COMMIT_SUBJECT=""
 
+// "temporary" workaround for aborting previous builds on PR update
```

Comment:
Please remove the word "temporary" from the comment.
You can add a `TODO` comment if you wish, to indicate that this is to be removed.

Reply:
Done